app/src/main/java/com/rockspin/androiddevtest/MyApplication.java

```diff
@@ -5,12 +5,23 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
+import java.util.List;
+
 import retrofit2.Retrofit;
 import retrofit2.converter.gson.GsonConverterFactory;
 
 public class MyApplication extends Application {
 
     private static APIService sAPIService;
+    private static MyApplication s_instance;
+    private List<CosmonautActivity> m_cosmonautActivities;
```

Comment:
Which activity do you mean? The MyApplication instance is static, due to the singleton pattern used.